### PR TITLE
Consolidated PR for bundle and build changes

### PR DESCRIPTION
**Pull request by itroyano**
Fixes: #43 #67 #68 

Signed-off-by: Igor Troyanovsky <itroyano@redhat.com>

### CODE REVIEW

File: .github/workflows/build-main.yml [hunk 1]
```diff
@@ -39,9 +39,6 @@ jobs:
       with:
         image: ${{ env.IMAGE_REGISTRY }}/operator-certification-operator-bundle
         tags: ${{ github.sha }} latest
```
Comment (bcrochet) [inline .github/workflows/build-main.yml:41]: Do we even care about the sha build now?

**Review by bcrochet — CHANGES_REQUESTED** [review]
Question about the sha tag.

File: .github/workflows/build-main.yml (hunk 1 above)
Comment (itroyano) [reply .github/workflows/build-main.yml:41]: It's there mostly for tracking daily development, and allowing someone to create an index pointing specifically to a bundle they'd like to test.
Not sure if that's a common use case though.

Comment (itroyano) [reply .github/workflows/build-main.yml:41]: @acornett21 @yashoza19 @spabba17 is there a use case where Preflight is run on an index that points to a specific bundle (not "latest")?
Or a catalog source created for such an index?

Comment (bcrochet) [reply .github/workflows/build-main.yml:41]: If someone wants to do that, they can push to their own quay.io repo. I don't see the utility of it honestly.

Comment (itroyano) [reply .github/workflows/build-main.yml:41]: in that case I'll get rid of it,  if no objections.

File: config/manifests/bases/operator-certification-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -37,15 +65,17 @@ spec:
...
-  - name: Operator Certification Operator
-    url: https://github.com/redhat-openshift-ecosystem/operator-certification-operator
+  - name: Operator Certification CI Pipeline
+    url: https://github.com/redhat-openshift-ecosystem/certification-releases/blob/main/4.9/ga/ci-pipeline.md
+  - name: Red Hat OpenShift Pipelines
+    url: https://github.com/openshift/tektoncd-operator
   maintainers:
-  - email: jomckenz@redhat.com
-    name: jomkz
+  - email: certification-operator-dev-team@redhat.com
```
Comment (itroyano) [inline config/manifests/bases/operator-certification-operator.clusterserviceversion.yaml:75]: pending - need to update this attribute with the actual email @jomkz created.

Comment (jomkz) [reply config/manifests/bases/operator-certification-operator.clusterserviceversion.yaml:75]: @itroyano I am anticipating that the request for the email alias will be approved and the email we will use will be:

`certification-operator-devel@redhat.com`

Comment (itroyano) [reply config/manifests/bases/operator-certification-operator.clusterserviceversion.yaml]: updated

**Review by bcrochet — APPROVED** [review]
/lgtm

**Pull request merged**